Utils.ps1

```diff
@@ -22,6 +22,9 @@ Set-Alias ?? Invoke-NullCoalescing -Force
 function Invoke-Utf8ConsoleCommand([ScriptBlock]$cmd) {
     $currentEncoding = [Console]::OutputEncoding
     try {
+        # A native executable that writes to stderr AND has its stderr redirected will generate non-terminating
+        # error records if the user has set $ErrorActionPreference to Stop. Override that value in this scope.
+        $ErrorActionPreference = 'Continue'

```

Comment:
That is correct PowerShell behavior.  Even if we redirect the error stream to $null, it still gets captures in $Error e.g.:
```
10:47> $error.Clear()
11:54> Write-Error "kaboom" 2>$null
12:48> $error
Write-Error "kaboom" 2>$null : kaboom
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException
```
What we want to avoid is the error record being displayed without evaluating $Error:
```
24:38> $error.Clear()
25:28> $ErrorActionPreference = 'Stop'
PS>Write-Error "kaboom" 2>$null
Write-Error "kaboom" 2>$null : kaboom
At line:1 char:1
+ Write-Error "kaboom" 2>$null
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException
```
Also, notice how my prompt function starts to fail, rendering `PS>` when I set $ErrorActionPreference to 'Stop'. This was done on master.  When I switch to this branch, the prompt function does not fail for me, so that is good!

RE $ErrorActionPreference scoping, yes PowerShell does a "copy-on-write" of variables assigned to in a lower scopes that have the same name as a variable in a higher dynamic scope.  This is why if you want to actually change the global value you have to use `$global:ErrorActionPreference`.